substrate/client/network/src/service/metrics.rs

```diff
@@ -225,6 +220,34 @@ impl Metrics {
 	}
 }
 
+/// PeerSet metrics.
+#[derive(Clone, Debug)]
+pub struct PeerSetMetrics {
```

Comment:
I would say it is `PeerStoreMetrics`, as it is common to all protocols. Currently we use "peer set" to refer to specific notifications protocol's peer set.

Reply:
Will rename thanks! 🙏